src/macro.c

```diff
@@ -379,25 +364,29 @@ void Macro::expand(OutBuffer *buf, size_t start, size_t *pend,
                 Macro *m = search(name, namelen);
                 if (m)
                 {
-#if 0
-                    if (m->textlen && m->text[0] == ' ')
-                    {   m->text++;
-                        m->textlen--;
-                    }
-#endif
                     if (m->inuse && marglen == 0)
                     {   // Remove macro invocation
                         buf->remove(u, v + 1 - u);
                         end -= v + 1 - u;
                     }
-                    else if (m->inuse && arglen == marglen && memcmp(arg, marg, arglen) == 0)
+                    else if (m->inuse &&
+                             ((arglen == marglen && memcmp(arg, marg, arglen) == 0) ||
+                              // marg is arg with blue paint added
+                              (arglen + 4 == marglen &&
+                               marg[0] == 0xFF &&
+                               marg[1] == '{' &&
+                               memcmp(arg, marg + 2, arglen) == 0 &&
+                               marg[marglen - 2] == 0xFF &&
+                               marg[marglen - 1] == '}'
+                              )
+                             )
+                            )
                     {   // Recursive expansion; just leave in place
-
+                        ;
```

Comment:
No, it won't. What it does do is make the line 'executable' so the -cov analyzer will indicate whether it was executed or not.